### PR TITLE
workbench beta phase complete

### PR DESCRIPTION
**Pull request by zkamvar**

### CODE REVIEW

**Review by elletjies — APPROVED** [review]

File: _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+In May 2023, we will transition all of our official lessons to use The Workbench.
+This transition will do two things:
+
+1. transform the file structure and syntax to The Workbench
+2. clean the Git history to accurately reflect lesson-specific commits.
+
+The transition process is automated in
+[carpentries/lesson-transition](https://github.com/carpentries/lesson-transition#readme)
+and lesson Maintainers will have the opportunity to audit the transformed
+lessons before it is transformed. A list of lessons and their transformation
```
Comment (tobyhodges) [inline _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md]:
```suggestion
and lesson Maintainers will have the opportunity to audit the transformed
lessons before the changes are finalised. A list of lessons and their transformation
```

File: _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+| -------------------------------------------------------- | ------------------------------------------------------ |
+| R for Social Scientists                                  | <https://datacarpentry.org/r-socialsci>                |
+| Introduction to Geospatial Raster and Vector Data with R | <https://datacarpentry.org/r-raster-vector-geospatial> |
+| Análisis y visualización de datos usando Python          | <https://datacarpentry.org/python-ecology-lesson-es>   |
+| Instructor Training                                      | <https://carpentries.github.io/instructor-training>    |
+
+
+## Workbench Transition to happen in May 2023
+
+In May 2023, we will transition all of our official lessons to use The Workbench.
```
Comment (tobyhodges) [inline _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md:39]:
```suggestion
The beta phase was a success,
and we can now begin final preparations for a full rollout of the new infrastructure.
In May 2023, we will transition all of our official lessons to use The Workbench.
```

File: _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+### Avoid making pull requests until May
+
+Because the transition will significantly change the folder structure and clean
+the Git history, it is important to avoid making pull requests to the lessons 
+in April. The reason for this is that after the transition, any old forks will
+be invalid and cannot be used to contribute. 
+
+If you have made a contribution to our lessons in the past and want to do so in
+the future, you will want to [delete and re-fork your fork of the lesson](https://carpentries.github.io/workbench/faq.html#update-fork-from-styles).
```
Comment (tobyhodges) [inline _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md]:
```suggestion
the future, you will need to [delete and re-fork your fork of the lesson](https://carpentries.github.io/workbench/faq.html#update-fork-from-styles).
```

File: _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+demonstrates what can happen if you do not delete your fork. 
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/9DVwy818MIE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+
+### Feature freeze until May
+
+Because we are now out of Beta, but not yet deployed to all of our lessons, The
+Workbench will be in a feature freeze until May. This means that if you request
+new features in The Workbench, you will have to wait until May to see the
```
Comment (tobyhodges) [inline _posts/2023/04/2023-04-04-workbench-beta-phase-complete.md]:
```suggestion
new features in The Workbench, you will have to wait until May at the earliest to see the
```

**Review by tobyhodges — APPROVED** [review]
Looks good to me. I have made only a few minor suggestions.

**Pull request merged**